### ISSUE TITLE
Lacking documentation

### ISSUE THREAD

**Issue by bphermansson** [cnlohr/tplink-raw-wifi#4]
I think this is an exciting project but hard to use as the documentation is lacking details. I've built the code as described, but whats next? Use the files in the dl folder for something?

**Comment by cnlohr** [comment]
Hmm... That's a decent point.  It is more like a bunch of disparate stuff used for dealing with raw-packet access on tplink 841N's (among others).  So you can get raw radio frames.  What were you looking to do?

**Comment by bphermansson** [comment]
I saw your video with the extreme low power Esp12 and I wanted to try it out. I have a router with Openwrt and some esps. I have built the code in this repo, but now I don't know what to do with it.

**Comment by cnlohr** [comment]
I guess I would need to understand more what your setup is and what you're shooting for?  What device do you want to use to receive the packets?

**Comment by bphermansson** [comment]
I just wanna do the same stuff as you did in your video, https://www.youtube.com/watch?v=I3lJWcRSlUA, but it's hard follow the instructions.

**Comment by cnlohr** [comment]
Do you have OpenWRT installed on a cheapie TPLink?

Do you have a build environment set up for OpenWRT?

**Comment by bphermansson** [comment]
Yes on both questions. I have a running OpenWrt and ran the commands in the Readme succesfullly (git clone --depth=1 https://git.lede-project.org/source.git, make).
